src/lib.rs

```diff
@@ -16,6 +16,8 @@ extern crate num_traits;
 #[cfg(feature = "serde")]
 #[macro_use]
 extern crate serde;
+#[cfg(test)]
+extern crate rand;
```

Comment:
Shouldn't this be in the `test` sub-module?

Reply:
Crates need to be in lib right?

Reply:
Good point!